### ISSUE TITLE
Layer normalization w/ or w/o affine parameters

### ISSUE THREAD

**Issue by MaroonAmor** [yash0307/RecallatK_surrogate#7]
Hi @yash0307,

I have another question about layer normalization applied in the model [here](https://github.com/yash0307/RecallatK_surrogate/blob/99262934ef09e1c7f2af832948c8899d93fbb6bb/src/netlib.py#L65).

The paper states that layer normalization is used similarly to the ProxyNCA++ method, where layer normalization does not have trainable affine parameters. However, in your code, by default, `torch.nn.LayerNorm(self.model.head.in_features)` will have trainable affine parameters enabled.

Is there any reason why the trainable affine parameters are enabled here? Have you noticed any performance between the cases w/ and w/o trainable affine parameters in layer normalization?

Thanks.

**Comment by yash0307** [comment]
Hi, we did not experiment with the trainable affine parameters disabled, so can not comment if it will make a difference in the performance. The comment in the paper is to highlight that other related papers such as ProxyNCA++ use layernorm as well.
